### DISCUSSION TITLE
bachlor文档类型，公式居中怎么修改。

### DISCUSSION THREAD

**Discussion by zjy-levi** [Aetf/xjtuthesis#47] [Q&A]
试了align 等方法，发现还是左对齐。救救

**Comment by Aetf** [comment]
没有任何细节的问题没有任何人可以帮助你的。

如果你可以提供一个可以复现的最小化例子，大家才知道你的问题可能是什么。

推荐阅读：https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md#精确地描述问题并言之有物

**Comment by Zipper-1** [comment]
公式请使用\begin{equation} \end{equation}块环绕
